tests/test_transforms/test_math/test_limit_transform.py

```diff
@@ -0,0 +1,129 @@
+    df2["segment"] = ["segment_2"] * periods
+    df2["target"] = np.random.uniform(-15, 5, size=periods)
+    df2.loc[10, "target"] = np.NaN
+    df2["target_no_change"] = df2["target"]
+    df2["non_target"] = np.random.uniform(-10, 0, size=periods)
+    df2["non_target_no_change"] = df2["non_target"]
+
+    df = pd.concat((df1, df2))
+    df = TSDataset.to_dataset(df)
+    tsds = TSDataset(df, freq="D")
+    return tsds
+
+
+@pytest.fixture
+def ts_check_limit_transform(random_seed) -> TSDataset:
+    periods = 4
+    df = pd.DataFrame({"timestamp": pd.date_range("2020-01-01", periods=periods)})
+    df["segment"] = ["segment_1"] * periods
+    df["target"] = np.array([1, 2, 3, 4])
+    df["target_no_change"] = np.array([1, 2, 3, 4])
+    df["target_transformed_1"] = np.array(
+        [2.000000165280742e-10, 4.000000330161484e-10, 6.000000494642226e-10, 8.000000658722967e-10]
+    )
+    df["target_transformed_2"] = np.array(
+        [-1.3862943610448906, -0.4054651080914978, 0.4054651080914977, 1.3862943610448906]
+    )
+    df["non_target"] = np.array([5, 6, 7, 8])
+    df["non_target_transformed_1"] = np.array(
+        [1.000000082240371e-09, 1.200000098568445e-09, 1.3999998928119147e-09, 1.599999909059989e-09]
+    )
+    df["non_target_transformed_2"] = np.array([-0.6931471805349453, 0.0, 0.6931471805349453, 1.6094379123541003])
+    df = TSDataset.to_dataset(df)
+    tsds = TSDataset(df, freq="D")
+    return tsds
+
+
+@pytest.mark.parametrize("lower_bound,upper_bound,n_test", [(-1e10, 1e10, "1"), (0, 5, "2")])
+def test_fit_transform_target(ts_check_limit_transform: TSDataset, lower_bound: float, upper_bound: float, n_test: str):
+    """Check forward scaled logit transform works correctly on target column"""
+    preprocess = LimitTransform(in_column="target", lower_bound=lower_bound, upper_bound=upper_bound)
+    result = preprocess.fit_transform(ts=ts_check_limit_transform).to_pandas()
+    np.testing.assert_array_almost_equal(
+        result["segment_1"]["target"], result["segment_1"]["target_transformed_" + n_test]
+    )
+
+
+@pytest.mark.parametrize("lower_bound,upper_bound,n_test", [(-1e10, 1e10, "1"), (3, 9, "2")])
+def test_fit_transform_non_target(
+    ts_check_limit_transform: TSDataset, lower_bound: float, upper_bound: float, n_test: str
+):
+    """Check forward scaled logit transform works correctly on nontarget column"""
+    preprocess = LimitTransform(in_column="non_target", lower_bound=lower_bound, upper_bound=upper_bound)
+    result = preprocess.fit_transform(ts=ts_check_limit_transform).to_pandas()
+    np.testing.assert_array_almost_equal(
+        result["segment_1"]["non_target"], result["segment_1"]["non_target_transformed_" + n_test]
+    )
+
+
+@pytest.mark.parametrize("lower_bound,upper_bound", [(-1e10, 1e10), (-15, 100)])
+def test_inverse_transform(ts_check_limit_transform: TSDataset, lower_bound: float, upper_bound: float):
```

Comment:
Here it is better to create copy of input fixture and compare it with dataframe after transform().inverse_transform()